en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-310-to-320.md

```diff
@@ -1356,7 +1356,20 @@ these alerts are persisted individually as well in tables specific for the type
 data related to alerts by dropping this table.
 
 ??? info "DB Scripts"
+     ```sql
      DROP TABLE APIMALLALERT;
+     ```
+
+!!! note
+    Only the `ANALYTICS_DB` need to be migrated and the other DBs (e.g. `DASHBOARD_DB`) can either be pointed to a new DB or be cleaned. If there are custom dashboards, those dashboards need to be exported and imported accordingly.
+
+    To export a dashboard, you need to login as an admin to dashboard listing page and select **Export** from the drop down menu (3 vertical dots) next to the dashboard name.
```

Comment:
shall we rename `drop down menu (3 vertical dots)` as `dashboard options menu` ?

Reply:
Fixed with 8144662